extensions/amp-analytics/0.1/analytics-root.js

```diff
@@ -279,6 +280,30 @@ export class AnalyticsRoot {
     });
   }
 
+  /**
+   * @param {string} selector DOM query selector.
+   * @return {!Promise<!Array<!Element>>} Element corresponding to the selector.
+   */
+  getElementsByScopedQuerySelectorAll_(selector) {
+    // Wait for document-ready to avoid false missed searches
+    return this.ampdoc.whenReady().then(() => {
+      const results = [];
+      const foundElements = scopedQuerySelectorAll(
+        dev().assertElement(this.ampdoc.getBody()),
+        selector
+      );
+
+      // Length is not supported in all browsers
+      if (foundElements && foundElements.length) {
```

Comment:
why don't we return `foundElements`?

Reply:
Oops, was previously doing a check there but it got removed. Thanks!

Reply:
there was a check there previously. thanks for the catch